moto/emr/__init__.py

```diff
@@ -6,7 +6,4 @@
 
 
 def mock_emr(func=None):
-    if func:
-        return MockAWS(emr_backends)(func)
-    else:
-        return MockAWS(emr_backends)
+    return MockAWS(emr_backends)
```

Comment:
Oh my. Sorry, this disabled the emr testing while I somehow thought it had just fixed my issue where `test_emr.test_create_job_flow_in_multiple_regions` would not pass. 

passes in CI now though I can't get it pass locally....